Automattic/jetpack
# WordAds: Add inline ads within post content

## Proposed changes:

* Adds inline ads within post content
* Adds option to enable/disable inline ads

![Screenshot 2024-05-01 at 5 56 40 PM](https://github.com/Automattic/jetpack/assets/1050616/a2dc9508-e5c6-452e-b114-2889229a6a0e)

![Screenshot 2024-05-01 at 5 56 57 PM](https://github.com/Automattic/jetpack/assets/1050616/7637d04b-4fda-49b2-8697-d3305e8c2c4e)

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Enabled WordAds on your test site
* Visit a post with a length that is at least 2x the window view height and includes multiple paragraphs of text
* Validate that an ad appears inline within the content
* You can append `wordads-logging=true` to the URL to enable detailed logging in the browser console
* Toggle the inline ad feature off
* Validate that inline ads no longer appear in the content
